validation_data/CSLC_Create_Validation_Table.ipynb

```diff
@@ -74,25 +74,27 @@
     "DATA_ROOT = 'science/SENTINEL1'\n",
```

Comment:
<p>You can actually delete/comment this. You only need to load the <code>validation_bursts_target_v{version_num}.csv</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OPERA-Cal-Val/calval-CSLC/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
csv file can be read into burst list. 
I added more corrections.
In run1_ALE_toggle_bursts_parallel.py, new CR data in Rosamond does not need solid earth tide correction.
Oversampling factor can be 32 or 64. 32 should work well with OPERA CSLC. Too large oversampling factor results in too large memory consumption and processing speed. See _Gisinger's 2021 paper:_ [10.1109/TGRS.2019.2961248](https://doi.org/10.1109/TGRS.2019.2961248)